bin/cml/pr.js

```diff
@@ -1,6 +1,6 @@
 const { options, handler } = require('./pr/create');
 
-exports.command = 'pr <glob path...>';
+exports.command = 'pr';
```

Comment:
Breaks backward compatibility because of exactly the same reason as it did at https://github.com/iterative/cml/pull/1073#discussion_r912862102 😅

Reply:
but this is just a docstring, right? doesn't affect implementation? Or are you telling me `yargs` works similar to `docopt` (docstring === implementation)?

Reply:
I'm telling you `yargs` et cetera...